airflow/providers/amazon/PROVIDERS_CHANGES_2020.05.10.md

```diff
@@ -0,0 +1,78 @@
+
+
+### Release 2020.05.10
```

Comment:
You are thinking we'd have one of these files per release we do?

(Rather than the one file per provider to which we add new sections?)

Reply:
Yes. I don't expect there will be many of those,

And we can always combine the older ones later if we find we have more of those files. It's just a bit easier to manage incremental releases and TOC if those are separate files - I do not have to parse the files to find out release version, just file name is enough.

Also it's kind a nice to see what were different releases just will looking at the directory.